velox/common/caching/SsdFile.h

```diff
@@ -314,6 +314,14 @@ class SsdFile {
     return writableRegions_.size();
   }
 
+  std::string getCheckpointFilePath() const {
```

Comment:
These can be private methods? Do we need them in test? Thanks!

Reply:
`getEvictLogFilePath()` is used in `testingTotalLogEvictionFilesSize()`